test/unit/domain/user/router/suite_test.go

```diff
@@ -0,0 +1,34 @@
+package router_test
+
+import (
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/lucky-pocket/luckyPocket-back/internal/app/user/delivery"
+	"github.com/lucky-pocket/luckyPocket-back/internal/global/validator"
+	"github.com/lucky-pocket/luckyPocket-back/internal/infra/web/http/filter"
+	"github.com/lucky-pocket/luckyPocket-back/test/mocks"
+	"github.com/stretchr/testify/suite"
+)
+
+type UserRouterTestSuite struct {
+	suite.Suite
+	engine          *gin.Engine
+	r               *delivery.UserRouter
+	mockUserUseCase *mocks.UserUseCase
+}
+
+func TestUserRouterSuite(t *testing.T) {
+	suite.Run(t, new(UserRouterTestSuite))
+}
+
+func (s *UserRouterTestSuite) SetupSuite() {
+	validator.Initialize()
+	s.mockUserUseCase = mocks.NewUserUseCase(s.T())
```

Comment:
```suggestion
	validator.Initialize()
	
	s.mockUserUseCase = mocks.NewUserUseCase(s.T())
```
이왕이면 띄어 놓는 게 좋을 것 같습니다!